config.toml.example

```diff
@@ -570,73 +567,76 @@ changelog-seen = 2
 
-# C compiler to be used to compiler C code. Note that the
+# C compiler to be used to compile C code. Note that the
 # default value is platform specific, and if not specified it may also depend on
 # what platform is crossing to what platform.
-#cc = "cc"
+# See `src/bootstrap/cc_detect.rs` for details.
+#cc = "cc" (path)
 
-# C++ compiler to be used to compiler C++ code (e.g. LLVM and our LLVM shims).
+# C++ compiler to be used to compile C++ code (e.g. LLVM and our LLVM shims).
 # This is only used for host targets.
-#cxx = "c++"
+# See `src/bootstrap/cc_detect.rs` for details.
+#cxx = "c++" (path)
 
 # Archiver to be used to assemble static libraries compiled from C/C++ code.
 # Note: an absolute path should be used, otherwise LLVM build will break.
-#ar = "ar"
+#ar = "ar" (path)
 
 # Ranlib to be used to assemble static libraries compiled from C/C++ code.
 # Note: an absolute path should be used, otherwise LLVM build will break.
-#ranlib = "ranlib"
+#ranlib = "ranlib" (path)
 
-# Linker to be used to link Rust code. Note that the
+# Linker to be used to bootstrap Rust code. Note that the
 # default value is platform specific, and if not specified it may also depend on
 # what platform is crossing to what platform.
 # Setting this will override the `use-lld` option for Rust code when targeting MSVC.
-#linker = "cc"
+#linker = "cc" (path)
 
 # Path to the `llvm-config` binary of the installation of a custom LLVM to link
 # against. Note that if this is specified we don't compile LLVM at all for this
 # target.
-#llvm-config = "../path/to/llvm/root/bin/llvm-config"
+#llvm-config = <none> (path)
 
 # Normally the build system can find LLVM's FileCheck utility, but if
 # not, you can specify an explicit file name for it.
-#llvm-filecheck = "/path/to/FileCheck"
+#llvm-filecheck = "<build-dir>/<target>/build/bin/FileCheck"
 
 # If this target is for Android, this option will be required to specify where
 # the NDK for the target lives. This is used to find the C compiler to link and
 # build native code.
-#android-ndk = "/path/to/ndk"
+# See `src/bootstrap/cc_detect.rs` for details.
+#android-ndk = <none> (path)
 
 # Build the sanitizer runtimes for this target.
 # This option will override the same option under [build] section.
-#sanitizers = false
+#sanitizers = build.sanitizers (bool)
 
 # Build the profiler runtime for this target(required when compiling with options that depend
 # on this runtime, such as `-C profile-generate` or `-Z instrument-coverage`).
 # This option will override the same option under [build] section.
-#profiler = false
+#profiler = build.profiler (bool)
 
 # Force static or dynamic linkage of the standard library for this target. If
 # this target is a host for rustc, this will also affect the linkage of the
 # compiler itself. This is useful for building rustc on targets that normally
 # only use static libraries. If unset, the target's default linkage is used.
-#crt-static = false
+#crt-static = <see above> (bool)
```

Comment:
Do you mean read the comment on this option? It seems like we should say something like "platform specific" rather than see above, which is pretty confusing to me at least.

Reply:
Sure thing, done.

Reply:
This doesn't seem done, unless I'm missing something.

Reply:
Fixed, thanks. Not sure if it just slipped my mind or if I lost it in git somewhere :shrug: